tests/visualization_test.py

```diff
@@ -149,6 +150,32 @@ def test_multi_objective_visualization_two_objectives():
     )
 
 
+def test_hypervolume_over_iterations_two_objectives():
+    evaluations = [
+        Evaluation(
+            objectives={"loss_1": 0.5 * i, "loss_2": -0.5 * i},
+            configuration={
+                "mlp_shape": 0.14054333845130684,
+                "optimizer": "Adam",
+                "batch_size": 160,
+            },
+            optimizer_info={"rung": -1},
+            user_info={},
+            settings={"fidelity": 2.5},
+        )
+        for i in range(5)
+    ]
+
+    evaluations_per_optimizer = {"test_optimizer": [evaluations]}
+
+    fig = hypervolume_over_iterations(
+        evaluations_per_optimizer,
+        objectives=(Objective("loss_1", False), Objective("loss_2", False)),
+        reference_point=[10, 0],
+    )
+    print(fig)
```

Comment:
Just out of curiosity, what is this print statement intended to do as part of the test?

Reply:
ah, left over after debugging session... I will remove it ;)